uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -1111,6 +1110,13 @@ define([
       function updateRequest() {
         var self = this;
 
+        if (self.isRole('manager')) {
+          //if manager has not changed the status then reset status
+          if (!self.request.status_id) {
+            self.request.status_id = self.statusBeforeEdit.value;
```

Comment:
for manager we filter the statuses (in `setStatuses` call which removes status which a manager can change in `requestStatuses` collection) and status before edit is set to null in html as its mapped to the model 
```
            <div class="chr_custom-select chr_custom-select--full">
              <select
                name="statusSelect"
                ng-options="status.value as status.label for status in $ctrl.requestStatuses"
                ng-model="$ctrl.request.status_id">
                <option value="">- select -</option>
              </select>
            </div>
```